.circleci/config.yml

```diff
@@ -79,6 +93,15 @@ commands:
           version: "16.1"
           device: "Apple TV"
 
+  install_rosetta_for_m1:
+    steps:
+      - run:
+          name: Install Rosetta for M1
+          command: |
+            if [ "${CIRCLE_JOB}" == "test-ios" ] || [ "${CIRCLE_JOB}" == "test-tvos" ]; then
```

Comment:
is this not needed for integration tests?

Reply:
Currently the integration tests are passing using the default medium resource class. My guess is since they use actual network calls the sequence of events execute more slowly and they don't experience the timeout issues in the functional tests. As the tests are passing, and running on an x86 chipset, there's no need to install Rosetta.